Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -642,7 +642,15 @@ private void RadioMenuSortDirection_Click(object sender, RoutedEventArgs e)
         private void AllView_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
         {
             tapDebounceTimer.Stop();
-            NavigationHelpers.OpenSelectedItems(ParentShellPageInstance, false);
+            
+            if (AppSettings.OpenFoldersNewTab)
+            {
+                NavigationHelpers.OpenPathInNewTab(((e.OriginalSource as FrameworkElement)?.DataContext as ListedItem).ItemPath);
+            }
+            else
+            {
+                NavigationHelpers.OpenSelectedItems(ParentShellPageInstance, false);
+            }
```

Comment:
This is only going to help when double clicking on an item, it's not going to help when pressing the enter key on an item.